src/command/build.rs

```diff
@@ -357,7 +354,7 @@ impl Build {
         Ok(())
     }
 
-    fn step_run_wasm_bindgen(&mut self, step: &Step) -> Result<(), Error> {
+    fn step_run_wasm_bindgen(&mut self, _step: &Step) -> Result<(), Error> {
```

Comment:
do we need to keep passing step if we don't use it?

Reply:
The step is passed through in a macro so is currently required to be present everywhere or nowhere, and it's still used in a few locations so I haven't deleted it yet.

Reply:
iirc step was introduced just to count the steps in the output- i imagine we could remove it with no loss - can you point to a place where it's load bearing?

Reply:
If that's the preference I can go ahead and remove it, I've been hoping to not have this PR expand in scope to include all the refactors related to the output, but if desired it can just keep going.

Reply:
so because master is going to be released soon and we don't want to release this in its current state- if we want to make progress, doing so on branch is a better idea. if you'd like, you can do the expansion of this PR in PRs branch off of this?

Reply:
Enough further work has happened on this branch now that I'm personally ok releasing it. I've pushed another commit which removes the `Step` type.